packages/document/docs/en/guide/advanced/browserslist.md

```diff
@@ -118,11 +118,10 @@ If you don't need to be compatible with IE 11 browsers, you can adjust Browsersl
 - Set to browsers that supports native ES Modules (recommended):
```

Comment:
@chenjiahan i was trying to work out where the default browser list was derived from
and it looks like this line was never updated with this PR

I think it should read something else after this update

Reply:
Could you create an issue to explain your requirements?